clusterloader2/testing/load/config.yaml

```diff
@@ -48,12 +48,38 @@
 {{$mediumDeploymentsPerNamespace := SubtractInt $mediumDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 {{$bigDeploymentsPerNamespace := SubtractInt $bigDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 
+# The minimal number of pods to be used to measure various things like
+# pod-startup-latency or scheduler-throughput. The purpose of it is to avoid
+# problems in small clusters where we wouldn't have enough samples (pods) to
+# measure things accurately.
+{{$MIN_PODS_IN_SMALL_CLUSTERS := 500}}
+
+# BEGIN scheduler-throughput section
+{{$totalSchedulerThroughputPods := MaxInt $MIN_PODS_IN_SMALL_CLUSTERS (MultiplyInt 2 .Nodes)}}
+{{$schedulerThroughputPodsPerDeployment := .Nodes}}
```

Comment:
nit: let's remove this constant - it doesn't bring much value and the names are terribly long.

Reply:
Ack, even though the name is long is descriptive and used in more than one place. IMHO, it makes the test easier to read and understand.